### PR TITLE
`loans` - remove collateral and confirm page

### PR DESCRIPTION
**Pull request by thedoublejay**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Ability to remove collateral
#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/EditCollateralScreen.tsx [hunk 1]
```diff
@@ -113,33 +112,16 @@ export function EditCollateralScreen ({
     dismissModal()
     const isConversionRequired = item.token.id === '0' ? new BigNumber(item.amount).gt(DFIToken.amount) : false
```
Comment (ivan-zynesis) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/EditCollateralScreen.tsx:113]: withdrawal case always no conversion require

Comment (thedoublejay) [reply mobile-app/app/screens/AppNavigator/screens/Loans/screens/EditCollateralScreen.tsx:113]: Yep there's no conversion for withdrawal. it's just add/remove are using the same component for collaterals

Comment (thedoublejay) [reply mobile-app/app/screens/AppNavigator/screens/Loans/screens/EditCollateralScreen.tsx:113]: For the check, it only happened twice, (EditCollateralScreen and on the bottom sheet token, when you select a token from the list)

File: mobile-app/app/screens/AppNavigator/screens/Loans/hooks/VaultShare.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import BigNumber from 'bignumber.js'
 
 export function useVaultShare (collateralAmount: BigNumber, factor: BigNumber, price: BigNumber, totalCollateralValue: BigNumber): BigNumber {
-  return new BigNumber(collateralAmount ?? 0).multipliedBy(price ?? 0).multipliedBy(factor ?? 0).dividedBy(totalCollateralValue ?? 1).times(100)
+  const vaultShare = new BigNumber(collateralAmount ?? 0).multipliedBy(price ?? 0).multipliedBy(factor ?? 0).dividedBy(totalCollateralValue ?? 1)
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Loans/hooks/VaultShare.ts:4]: i think none of the params are optional, does `??` checks for anything?

Comment (thedoublejay) [reply mobile-app/app/screens/AppNavigator/screens/Loans/hooks/VaultShare.ts:4]: Oh yeah I can remove it. It's just for an earlier version I hadn't updated

Comment (thedoublejay) [reply mobile-app/app/screens/AppNavigator/screens/Loans/hooks/VaultShare.ts]: updated

**Review by kyleleow — APPROVED** [review]

**Pull request merged**